Detectors/TRD/base/src/TrackletTransformer.cxx

```diff
@@ -123,7 +102,8 @@ CalibratedTracklet TrackletTransformer::transformTracklet(Tracklet64 tracklet)
   auto slope = tracklet.getSlopeBinSigned();
 
   // calculate raw local chamber space point
-  loadPadPlane(detector);
+  mPadPlane = mGeo->getPadPlane(detector);
```

Comment:
I guess this no longer needs to be a member variable.
```suggestion
  auto padPlane = mGeo->getPadPlane(detector);
```

Reply:
Well it is used in three other functions which are called from there, so one would need to pass it along every time. I would leave it as it is for now.